evo_researcher/benchmark/benchmark.py

```diff
@@ -147,7 +183,7 @@ def get_markets_summary(self) -> t.Dict[str, t.List[str]]:
         }
         for model_type in self.predictions.keys():
             markets_summary[f"{model_type} p_yes"] = [
-                p.p_yes for p in self.predictions[model_type]
+                p.p_yes for p in self.predictions[model_type].values()
```

Comment:
This may give the wrong order of predictions because predictions in `self.predictions[model_type]` doesn't have to match with `self.markets`.

```suggestion
                markets_summary[f"{model_type} p_yes"] = [
                (self.predictions[model_type][q.question].p_yes for q in self.markets
            ]
```

Reply:
Good spot!